style.css

```diff
@@ -22,15 +22,25 @@ body {
 
 /* ----------------------------------------------POP UP display ----------------------------------------------------------- */
 .popup-container {
-  background: #fff;
+  display: none;
+  position: absolute;
+  left: 0;
+  top: 0;
+  z-index: 10;
+  background-color: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 16px;
+
+  /* background: #fff;
   border: 1px solid #ebecf0;
   position: absolute;
   z-index: 10;
   left: 20%;
   top: 0;
   display: none;
   box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
-  border-radius: 16px;
+  border-radius: 16px; */
```

Comment:
- [ ] Kindly make sure to remove these lines of code you just commented-out to make your project more professional and clean :recycle: